static/app/views/performance/landing/index.tsx

```diff
@@ -69,7 +72,33 @@ export function PerformanceLanding(props: Props) {
 
   const {teams, initiallyLoaded} = useTeams({provideUserTeams: true});
 
-  const currentLandingDisplay = getCurrentLandingDisplay(location, projects, eventView);
+  const hasMounted = useRef(false);
+  const paramLandingDisplay = getLandingDisplayFromParam(location);
```

Comment:
This and the mounted check is so that linking to specific still works (it will mount the param on load, otherwise ignore it)